src/oidcc_app.erl

```diff
@@ -0,0 +1,61 @@
+-module(oidcc_app).
+
+-export([start/2]).
+-export([stop/1]).
+-export([init/1]).
+-export([handle_call/3]).
+-export([handle_cast/2]).
+-export([handle_info/2]).
+-export([terminate/2]).
+-export([httpc_profile/0]).
+
+-behaviour(application).
+-behaviour(gen_server).
+
+%% @private
+httpc_profile() ->
+    oidcc.
+
+%% Application Callbacks
+
+%% @private
+start(_StartType, StartArgs) ->
+    gen_server:start_link(oidcc_app, StartArgs, []).
+
+%% @private
+stop(_State) ->
+    ok.
+
+%% GenServer Callbacks
+%% @private
+init(_Args) ->
+    try
+        inets:start(httpc, [{profile, httpc_profile()}])
+    catch
+        error:{already_started, _} -> ok
+    end,
+
+    % disable keep-alive
+    httpc:set_options(
+        [
+            {pipeline_timeout, 0},
```

Comment:
We can drop all of this with the new approach, right?

Reply:
:+1: yep, forgot about that